tools/dify_neo4j.bak/tools/neo4j_query.py

```
@@ -0,0 +1,30 @@
+            return
+
+        try:
+            driver = GraphDatabase.driver(url, auth=basic_auth(username, password))
+            with driver.session() as session:
+                result = session.run(query)
+                records = [record.data() for record in result]
+            driver.close()
+
+            yield self.create_json_message({"results": records})
+            yield self.create_text_message(str(records))
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)